aim/sdk/objects/audio.py

```diff
@@ -0,0 +1,96 @@
+import io
+import logging
+
+from aim.sdk.num_utils import inst_has_typename
+from aim.sdk.objects.io import wavfile
+from aim.storage.object import CustomObject
+from aim.storage.types import BLOB
+
+logger = logging.getLogger(__name__)
+
+
+@CustomObject.alias('aim.audio')
+class Audio(CustomObject):
+    AIM_NAME = 'aim.audio'
+
+    # supported audio formats
+    UNKNOWN = 0
+    MP3 = 1
+    WAV = 2
+    FLAC = 3
+
+    __audio_format_map = {
```

Comment:
let's use strings as inputs. we still have to do validation of formats, but users will prefer not to learn new naming, enums, etc.
besides, the format we'll store and later read from storage will be string.